build.xml

```diff
@@ -63,9 +63,11 @@
                     <pathelement path="${build-debug}"/>
                 </classpath>
                 <formatter type="plain"/>
-                <test name="TestCase" haltonfailure="no" outfile="test-report/result">
+                <batchtest haltonfailure="no" todir="test-report">
```

Comment:
Now by using batchtest we can run multiple matching tests and we add TestCase2 accordingly.

Reply:
Ideally we should hv TestCase1...N to cover various edge cases and enjoy high coverage ratio.